pkg/server/server_test.go

```diff
@@ -149,6 +149,9 @@ func TestCheckDoesNotThrowBecauseDirectTupleWasFound(t *testing.T) {
 			ResolveNodeLimit: 25,
 		},
 		checkResolver: graph.NewLocalChecker(storage.NewContextWrapper(mockDatastore), 100),
+		typesystemResolver: typesystem.MemoizedTypesystemResolverFunc(
+			typesystem.NewTypesystemResolver(mockDatastore),
+		),
 	}
```

Comment:
please change this so that it uses `server.New( ... )` and we don't have to manually specify these fields

Reply:
See https://github.com/openfga/openfga/pull/623